.github/workflows/build.yml

```diff
@@ -0,0 +1,108 @@
+name: Build
+
+on: 
+  push:
+    branches:
+      - 'master'
+  pull_request:
+
+
+jobs:
+  Build:
+    name: Build appimagetool
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        include:
+          - ARCH: 'x86_64'
+            docker-container: 'appimagebuild'
+          - ARCH: 'i386'
+            docker-container: 'appimagebuild-i386'
+          - ARCH: 'armhf'
+            docker-container: 'appimagebuild-armhf-cross'
+          - ARCH: 'aarch64'
+            docker-container: 'appimagebuild-aarch64-cross'
+      fail-fast: false
+    env:
+      DOCKER_IMAGE: quay.io/appimage/${{ matrix.docker-container }}
+      ARCH: ${{ matrix.ARCH }}
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Install dependencies for ARM and AARCH64
+      if: endsWith(matrix.docker-container, 'cross') 
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install binfmt-support qemu-user-static
+
+    - name: Build 
+      shell: bash
+      run: |
+        if [[ "$ARCH" == "i386" ]]; then
+          sudo dpkg --add-architecture i386
+          sudo apt-get update
+        fi
+        bash -ex travis/travis-build.sh
+
+    - name: List files
+      run: |
+        ls -lh build/out/*
+
+    - name: Upload artifact
+      uses: actions/upload-artifact@v1.0.0
+      with:
+        # Artifact name
+        name: appimagetool-${{ matrix.ARCH }}.AppImage.zip
+        path: build/out
+
+
+  Release:
+    needs: [Build]
+    runs-on: ubuntu-latest
+ 
+    steps:
+    - uses: actions/download-artifact@v1
+      with:
+        name: appimagetool-x86_64.AppImage.zip
+    - uses: actions/download-artifact@v1
+      with:
+        name: appimagetool-i386.AppImage.zip
+    - uses: actions/download-artifact@v1
+      with:
+        name: appimagetool-armhf.AppImage.zip
+    - uses: actions/download-artifact@v1
+      with:
+        name: appimagetool-aarch64.AppImage.zip
+
+    - name: Continuous Release
+      uses: marvinpinto/action-automatic-releases@latest
```

Comment:
Please use pyuploadtool here, too, which is designed to work with our continuous release scheme. You don't need the sketchy `if` below then either.
pyuploadtool doesn't even hardcode the default branch name, but fetches it from the GitHub API.
You can likely dump the "tagged release" build as well, pyuploadtool differentiates between continuous and release builds using the same assumptions we've had in @probonopd's uploadtool (I reverse engineered that code a bit).

Reply:
Perfect. I will use `pyuploadtool`